src/main/target/CRASHTESTF7/config.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
```

Comment:
commenting out definitions gets pretty ambigious without commenting why these got disabled. please clean up or add comments

Reply:
Yes Kiao if you could make comments why this is commented out or just remove it then I'll be down to get this all merged into the code. Thanks for your work buddy!